accounts/forms.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+
+class SignupForm(UserCreationForm):
+
+    class Meta:
+        model = get_user_model()#CustomUserモデル、DjangoデフォルトのUserモデルを問わず、使用しているUserモデル自体を返してくれる
```

Comment:
```suggestion
User = get_user_model()

class SignupForm(UserCreationForm):

    class Meta:
        model = User
```
上記のように書くのが一般的なので、このように変更お願いします。